manual/program/proposal_guidelines.rst

```diff
@@ -0,0 +1,56 @@
+Proposal Submission Guidelines
+==============================
+
+.. image:: tiptap.gif
+  :width: 400
+  :alt: Meme cat writing a SciPy proposal
+
+Here are the instructions and tips for submitting a proposal to SciPy.
+
+Instructions
+------------
+
+.. image:: tiptappy.gif
+  :width: 400
+  :alt: Another meme cat writing a SciPy proposal
+
+**Abstract**
+
+Your Abstract will appear in the online schedule and give attendees a sense of your talk. This should be around 100 words or less.
+
+**Description**
+
+Your placement in the program will be based on reviews of your description. This should be a roughly 500-word outline of your presentation. This outline should concisely describe software of interest to the SciPy community, tools or techniques for more effective computing, or how scientific Python was applied to solve a research problem. A traditional background/motivation, methods, results, and conclusion structure is encouraged but not required. Links to project websites, source code repositories, figures, full papers, and evidence of public speaking ability are encouraged.
+
+**Format**
+
+You must choose whether to submit as a talk or a poster. Talks that are not selected will automatically be considered for a poster slot. Please note that a virtual-only poster is possible.
+
+**Paper**
+
+The form allows you to upload a paper. This is optional. You do not need to upload a paper in order to submit a talk or poster.
+
+Tips for Submitting a Proposal
+------------------------------
+
+.. image:: tippytappy.gif
+  :width: 400
+  :alt: Yet another meme cat writing a SciPy proposal
+
+The SciPy Conference is in awe of the work that is being done in the community. We receive many interesting and thought-provoking proposals but we have a limited number of spaces. Please take a look at our tips below to improve your chances of having a talk or poster accepted by the conference. In the unfortunate event that your proposal is not accepted, please keep in mind that you are welcome to give a lightning talk, book a room for a Birds of a Feather discussion, or talk to the Program Committee about displaying your work as a poster in lieu of a talk.
+
+- Submit your proposal early.
+
+- In your abstract, be sure to include answers to some basic questions: Who is the intended audience for your talk? What, specifically, will attendees learn from your talk?
+
+- Ensure that your talk will be relevant to a broad range of people. If your talk is on a particular Python package or piece of software, it should useful to more than a niche group.
+
+- Include links to source code, articles, blog posts, or other writing that adds context to the presentation.
+
+- If you've given a talk, tutorial, or other presentation before, include that information as well as a link to slides or a video if they're available.
```

Comment:
Do we want to recommend that they put these links in the "notes" section. It seems a little bit random to include links to other scipy talks if they aren't related, and I don't think this should be published in the program.

Reply:
Sure that works for me!